README.md

```diff
@@ -2,55 +2,34 @@
 
 # Project Name
 
-> Description the project.
+> This project is a simple to do list of tasks. Built using webpack.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- webpack
+- Javascript
+- HTML & CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
 ### Usage
-
-### Run tests
-
-### Deployment
-
+1. Clone repo
+2. Install all dependencies
+3. use <npm run start> to run the local server.
```

Comment:
- [ ] Please it will be nice to add explicit steps on how to clone and run your app locally, as this will be of help to non technical users.